src/Http/Http.Extensions/gen/StaticRouteHandlerModel/StaticRouteHandlerModel.Emitter.cs

```diff
@@ -108,14 +116,12 @@ private static string EmitResponseWritingCall(this Endpoint endpoint)
      * can be used to reduce the boxing that happens at runtime when constructing
      * the context object.
      */
-    public static string EmitFilteredRequestHandler(this Endpoint endpoint)
+    public static string EmitFilteredRequestHandler(this Endpoint _)
     {
-        var argumentList = endpoint.Parameters.Length == 0 ? string.Empty : $", {endpoint.EmitArgumentList()}";
```

Comment:
I assume this is intentionally since you're anticipating that I'll push this forward with my changes to introduce the variadic `InvocationFilterContext` overloads.

Reply:
Yep, for now we didn't need the argument list, but I left it as an extension method even though technically it doesn't use any of the state from the target object.